apps/expo/package.json

```diff
@@ -38,6 +38,7 @@
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.1",
+    "react-native-vector-icons": "^10.0.2",
```

Comment:
can you remove this dependency -- for now just use placeholder text for icons, since we'll be using a different set of icons for our design system